openstack/ironic/templates/seed.yaml

```diff
@@ -218,6 +229,14 @@ spec:
         role: cloud_baremetal_admin
       - user: ipmi_exporter@Default
         role: cloud_baremetal_admin
+    - name: BaremetalMonitoring
+      roles:
+      - user: image-build@Default
+        role: cloud_image_admin
+      - group: CCADMIN_CLOUD_ADMINS
+        role: admin
+      - group: CCADMIN_CLOUD_ADMINS
+        role: member
```

Comment:
I think admin includes the scopes of members

Reply:
No, it is missing. When clicking on server overview I am getting ```You are not authorized to view this page. Please check (role assignments) if you have one of the following roles: cloud_compute_admin, compute_admin, member, Member, compute_viewer.```